tests/test_install_claude_action_secrets.sh

```
@@ -0,0 +1,309 @@
+  REPO_ROOT="$repo_root"
+
+  local out
+  if [[ "$mode" == "missing" ]]; then
+    # gh が PATH にあっても無くても、必ず not-found 扱いにする
+    out=$(
+      command() {
+        if [[ "$1" == "-v" && "$2" == "gh" ]]; then
+          return 1
+        fi
+        builtin command "$@"
+      }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)